ios/RTCPjSip/PjSipAccount.m

```diff
@@ -116,6 +116,36 @@ - (id)initWithConfig:(NSDictionary *)config {
                 }
             }
         }
+        // TURN settings
+        NSDictionary *turnConfiguration = config[@"turn"];
+        if (turnConfiguration) {
+            NSString *turnServer = turnConfiguration[@"server"];
```

Comment:
Thanks for the PR!
Is it possible to split the code to avoid copy-paste for pjsua_turn_config struct creation ?